examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+
+#### Cloudretro Coordinator deployment & service
+
+Now, we will need the container-image for CloudRetro. The simplest way is to download a very lightly modified version (for easier handling in k8s enviroment), but you can also build your own image [here](https://github.com/giongto35/cloud-game).
+
+After you got your image, we can jump right into deployments and pods. For this, I am using the image I`ve modified for this purpose (which is far from flawless, but it works).
+The following yaml file will make a deployment for the Coordinator, best to leave the scaling at 1 (TODO, worker sign in problems). By default, the Coordinator HTTP service will listen at port 8000, if you changed this in the config, make sure to change it here as well.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coordinator-deployment
+  namespace: cloudretro
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: coordinator
+  template:
+    metadata:
+      labels:
+        app: coordinator
+    spec:
+      containers:
+      - name: coordinator
+        image: valniae/snekyrepo:cloudimage4
+        volumeMounts:
+        - name: cloudretro-confmap-vol
+          mountPath: "/usr/local/share/cloud-game/configs/"
+          readOnly: true
+        command: ["coordinator"]
+        args: ["--v=5"]
+        ports:
+        - containerPort: 8000
+      volumes:
+      - name: cloudretro-confmap-vol
+        configMap:
+          name: cloudretro-config
+EOF
+```
+
+This should leave us with a working pod of coordinator:
+
+```console
+kubectl get pods -n cloudretro
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+coordinator-deployment-54hidoyoureadthis-qpnsx   1/1     Running   0          3m23s
+```
+
+Now that we have a running deployment of Coordinator(s), we will need a service for them. So in case a pod would die for whatever reason, the target address wouldn`t change for the users. In case you require, use a firewall-open TCP port for Nodeport, don`t forget to uncomment.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: coordinator-lb-svc
+  namespace: cloudretro
+spec:
+  selector:
+    app: coordinator
+  ports:
+    - port: 8000
+      targetPort: 8000
+      #nodePort: 30001
+  type: LoadBalancer
+EOF
+```
+
+Now we should be having a fancy LoadBalancer service; whatever NodePort you`ve decided on (if).
+
+```console
+kubectl get services -n cloudretro
+
+NAME                 TYPE           CLUSTER-IP            EXTERNAL-IP            PORT(S)          AGE
+coordinator-lb-svc   LoadBalancer   <YOUR_C_CLUSTER_IP>   <YOUR_C_EXTERNAL_IP>   8000:30618/TCP   2m29s
+```
+
+Now you should be able to successfully connect to <YOUR_EXTERNAL_IP>:8000 service, leaving you on a nice, blank controller page. Have no worries, without workers we didn`t expect you to play any games on it.
```

Comment:
you must walk the users through the way to identify the EXTERNAL_IP here: please include CLI calls (e.g., "kubectl get ...") they can use to load their EXTERNAL_IP into an environment variable and then use that environment var in the comand lines below. please refer to the rest of the examples on how to do that